e2e/package.json

```diff
@@ -12,14 +12,13 @@
   },
   "homepage": "https://github.com/rocket-connect/graphql-debugger#readme",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc -b",
```

Comment:
Just to clarify what the -b is for here?

Reply:
That's the `build` option - you need it to ensure that all dependencies below this package are built. I found that turbo wasn't running anything in a particular order so it required this